### DISCUSSION TITLE
2005/05/07/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95-2

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#23] [Announcements]
# 2005/05/07/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95-2

为了方便的对内容进行更新，已把此篇文章转到了新地址：

https://blog.qizhen.xyz/2005/05/07/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95-2

**Comment by ruanqizhen** [comment]
ruby.chen
2005年10月15日 13:20 编辑
我哭泣了!!!!

回复
ruby.chen
2005年10月15日 13:20 编辑
为什么我进不去我的空间啊!!!!

回复
多多
2005年10月6日 16:54 编辑
呵呵，我来也

回复
alex
2005年09月9日 23:36 编辑
谢谢咯

回复
fan
2005年08月25日 14:29 编辑
留个脚印

回复
匕禾页
2005年08月3日 23:22 编辑
你真是个热心的人，谢谢啦！

回复
健
2005年07月31日 23:09 编辑
太好了,佩服~~

回复
Unknown
2005年07月25日 21:03 编辑
字有点小,看不清耶,能不能拜托弄大点呢?

回复
rOckbeAr
2005年06月5日 11:45 编辑
十分十分的感谢

回复
Radon
2005年06月3日 12:13 编辑
厉害哦，真的很专业耶……我那个就不行了，空白的要命！有空再学习一下……呵呵

回复
Unknown
2005年05月27日 18:39 编辑
你好专业哦，我正在建设中，一定要用IE吗？

回复
天水崖
2005年05月26日 08:54 编辑
学一下

回复
Jessica
2005年05月15日 11:31 编辑
牛X 学习一下。

回复
jean
2005年05月10日 10:13 编辑
you are the first one who come to my MSN space,welcome!And ,i will see you .

回复
妖。
2005年05月8日 22:09 编辑
哈哈….我也来你这啦~~~~很高兴认识你

回复
